common/djangoapps/student/models.py

```diff
@@ -1570,10 +1571,13 @@ def refundable(self, user_already_has_certs_for=None):
 
         # If it is after the refundable cutoff date they should not be refunded.
         refund_cutoff_date = self.refund_cutoff_date()
-        if refund_cutoff_date and datetime.now(UTC) > refund_cutoff_date:
+        # refund_cuttoff_date will be none if there is no order. If there is no order return False
+        if not refund_cutoff_date:
```

Comment:
if you're checking for `None`, specifically check `if refund_cutoff_date is None`